build-hugo-docs.py

```diff
@@ -119,6 +119,7 @@
     fh.write(test_out)
 
 # Run docs/hugo/docs-uan/bin/build.sh
+subprocess.call(["python", THIS_DIR + "/gen-hugo-toc-json.py"])
```

Comment:
This call needs to move to the docs/hugo/docs-uan/lib/content_prep script as the hugo_toc.json used in the build of the hugo docs resides in a checked out version (based on tags in release_list.yaml) and not in the main branch.  We never build docs content in the main branch.  It needs to run in each cloned repo.  The convert-docs-to-hugo.sh script runs in a container for each checked out release we build and runs the functions in lib/content_prep.  The gen-hugo-toc-json.py script will also need to be in the /docs/hugo/docs-uan/lib directory.

Also, "python" should probably be "python3".